terraform/README.md

```diff
@@ -0,0 +1,68 @@
+# LMNH AWS Cloud Architecture
+This folder contains the terraform config to provision the following cloud architecture displayed in _Figure 1_
+
+![LMNH-cloud-architecture-diagram](../assets/starwatch_architecture_diagram.png)
```

Comment:
Link to diagram is nice

Reply:
Good link to the architecture diagram, allowing the reader to properly reference what you're talking about.